server.js

```diff
@@ -14,7 +14,8 @@ const modules = [
     'rules/jar',
     'card',
     'help',
-    'hangman'
+    'hangman',
+    'whatsInStandard'
```

Comment:
I'd prefer to keep the module filenames the same as the main module commands. So instead of calling the file `whatsInStandard.js`, I'd simply call it `standard.js`.